### ISSUE TITLE
False Positive if using CloudFlare

### ISSUE THREAD

**Issue by ItzNotABug** [afollestad/nock-nock#52]
*Please consider making a Pull Request if you are capable of doing so.*

**App Version:**

0.8.8
 
**Affected Device(s):**
 
All, probably..
 
**Describe the Bug:**
If the site is using cloudflare, and there is a wait time shown by cloudflare (5 seconds wait screen) then the app logs it as an Error and shows a 503 Error.

**Expected Behavior:**
Should wait for the expected time, then check if the site is working or not..

**Comment by afollestad** [comment]
503 means service unavailable, that is not a recoverable status code.

**Comment by ItzNotABug** [comment]
but that is returned even while the site is active behind cloudflare..

**Comment by afollestad** [comment]
That's wrong behavior though. They shouldn't be returning that status code unless something is down.

**Comment by Akito13** [comment]
@ItzNotABug Your name fits this issue.
